ios/src/TGGeoPoint.h

```diff
@@ -0,0 +1,27 @@
+//
+//  TGGeoPoint.h
+//  tangram
+//
+//  Created by Karim Naaji on 10/27/16.
+//
+//
+
+#ifndef TGGeoPoint_h
+#define TGGeoPoint_h
```

Comment:
As far as I know, `#pragma once` works in C for every compiler I've ever heard of - have you found that to be untrue?